docs/changelog.rst

```diff
@@ -16,6 +16,7 @@ Changelog
 * :feature:`2844` Premium users will now be able to fetch their Uniswap v3 swaps.
 * :feature:`2893` Users can now see if any of their addresses have CVX available to claim from the ConvexFinance airdrop.
 * :bug:`2850` User will now see a consistent naming of exchanges across the application.
+* :bug:`367` Requests to external services won't make rotki hang for long periods.
```

Comment:
This does not really need a changelog. But if it does, then this one sounds like every request to external service may hang for a long period. Perhaps change to: 
```suggestion
* :bug:`367` Fixed edge cases where some tasks would run for hours due to the absence of timeouts.
```